script.js

```diff
@@ -74,6 +84,26 @@
     this.message = options.message || {};
     var renderedCallback = options.onButtonRendered;
 
+    this.customSettings = options.customElementSettings || null;
+    if (this.customSettings) {
+      while (this.customElementGroup.firstChild) {
+        this.customElementGroup.removeChild(this.customElementGroup.firstChild);
+      }
+      // Focusable should be true when it is defaultFocus.
+      this.customSettings.focusable = this.customSettings.defaultFocus ?
```

Comment:
Please use `this.customSettings.focusable |= this.customSettings.defaultFocus;` or `this.customSettings.focusable = this.customSettings.defaultFocus || this.customSettings.focusable;`